docs/Product-Strategy/Roadmap-and-Backlog.md

```diff
@@ -1,32 +1,32 @@
-_updated May 2021_
+_updated June 2021_
 # Product Roadmap
-Our [roadmap](https://app.mural.co/t/officeoffamilyassistance2744/m/officeoffamilyassistance2744/1617994644349/6047bad9ed365b22986e52f5d9f01aac32286ae9) :lock:represents our latest thinking about the order in which we’ll tackle the various pieces of the overarching problem.
+Our [roadmap](https://app.mural.co/t/officeoffamilyassistance2744/m/officeoffamilyassistance2744/1617994644349/6047bad9ed365b22986e52f5d9f01aac32286ae9) :lock: represents our latest thinking about the order in which we’ll tackle the various pieces of the overarching problem.
 
 We consider ATO, Release 1 and Release 2 commitments that the team will deliver on at this time. There might be a few shifts in approach, timing or scope, but in general, these outcomes will be worked on by the team. Metrics for success will be added as they are defined for releases.
 
 Beyond that, we're still discovering and planning on what best serves our users. This doc will continue to be updated as we make decisions and scope releases. 
 
-## ATO  
+## ATO, _3 year ATO granted on May 20, 2021_  
 Value Delivered: Get approval for the authority to operate and create a production environment. 
 
 | Outcome | Status | 
 | -------- | ------- | 
 | User can log in using login.gov | Complete
 | Users with appropriate privileges can manage users | Complete
 | Users can upload data files by section and quarter | Complete
-| Users with appropriate privileges can download files that were previously uploaded     | In Progress
-|Create production environment | Will happen after ATO
+| Users with appropriate privileges can download files that were previously uploaded     | Complete
+|Create production environment | In Progress
 
 ## Release 1 (fka OFA MVP) Scope
-Value Delivered: ACF employees can use a secure method to log into the system; this would require a PIV card.
+Value Delivered: ACF employees can use a secure method to log into the system; this would require a PIV/CAC card.
```

Comment:
Do we want to mention anything about the implementation of secret key prevention? it's definitely valuable in that this is providing reassurance about the security of TDP, but I understand if we want to focus primarily on user-facing outcomes.

Reply:
Yep, absolutely. I'll add some language in.